### PR TITLE
ViewHelper: Introduce animated view presets.

### PR DESCRIPTION
**Pull request by Mugen87**
This PR enhances the new `ViewHelper` class by introducing animated view presets. If you now click on of the the visualized axes, the camera will be animated to a pre-defined view.

### CODE REVIEW

**Comment by Mugen87** [comment]
https://raw.githack.com/Mugen87/three.js/dev43/editor/index.html

File: editor/js/Viewport.js [hunk 1]
```diff
@@ -220,6 +226,14 @@ function Viewport( editor ) {
 
 	}
 
+	function isUIInteraction( event ) {
```
Comment (mrdoob) [inline editor/js/Viewport.js:229]: Have you tried adding a invisible dom element to `VisualHelper` that listens for mouse events and has a `event.stopPropagation()`?

Comment (mrdoob) [reply editor/js/Viewport.js:229]: Basically, to let the browser decide where the mouse event went instead of doing it yourself.

Comment (Mugen87) [reply editor/js/Viewport.js:229]: > Have you tried adding a invisible dom element to VisualHelper that listens for mouse events and has a event.stopPropagation()?

No, not yet^^. Let me give this a try 👍 .

Comment (Mugen87) [reply editor/js/Viewport.js]: Okay, I've updated the PR. Using an invisible DOM element means less changes to `Viewport`. Minor disadvantage. Orbiting does now not work anymore if you move with pressed mouse button over the helper. However, this is also true for the toolbar.

Comment (mrdoob) [reply editor/js/Viewport.js]: Nice!

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**